app/views/layouts/application.html.erb

```diff
@@ -11,8 +11,15 @@
   </head>
 
   <body>
-    <% if notice %> <p class="notice"><%= notice %></p> <% end %>
-    <% if alert %><p class="alert"><%= alert %></p> <% end %>
+    <% flash.each do |type, msg| %>
+      <% if type == 'success' %>
+        <div class="alert alert-success">
+          <%= msg %>
+        </div>
+      <% end %>
+    <% end %>
```

Comment:
I like the way you managed the different flashes. This way, we don't have to worry too much about the names and keep it drier. I'll delete my specific ones to use this instead in my next feature. Thanks!